workers/cpp_physics/physics.cc

```diff
@@ -0,0 +1,104 @@
+        std::cout << std::endl;
+    };
+
+    if (argc != 4) {
+        print_usage();
+        return kErrorExitStatus;
+    }
+
+    worker::EntityId entityId = 1;
+
+    worker::ConnectionParameters parameters;
+    parameters.WorkerType = "physics";
+    parameters.Network.ConnectionType = worker::NetworkConnectionType::kTcp;
+    parameters.Network.UseExternalIp = false;
+    worker::Connection connection = worker::Connection::ConnectAsync(
+        ComponentRegistry{}, std::string{argv[1]}, atoi(argv[2]), argv[3], parameters).Get();
+
+    if (connection.GetConnectionStatusCode() != worker::ConnectionStatusCode::kSuccess) {
+        std::cerr << "Failed to connection the receptionist." << std::endl;
+        std::cerr << "Reason: " << connection.GetConnectionStatusDetailString() << std::endl;
+        return kErrorExitStatus;
+    }
+    std::cout << "Successfully connected using the Receptionist" << std::endl;
+
+    worker::View view{ComponentRegistry{}};
+    bool is_connected = true;
+
+    view.OnDisconnect([&](const worker::DisconnectOp& op) {
+        std::cerr << "[disconnect] " << op.Reason << std::endl;
+        is_connected = false;
+    });
+    view.OnLogMessage([&](const worker::LogMessageOp& op) {
+        if (op.Level == worker::LogLevel::kFatal) {
+            std::cerr << "Fatal error: " << op.Message << std::endl;
+            is_connected = false;
+        } else {
+            std::cout << "[remote] " << op.Message << std::endl;
+        }
+    });
+    view.OnCommandRequest<sample::Login::Commands::TakeControl>([&](const worker::CommandRequestOp<sample::Login::Commands::TakeControl>& op) {
+        connection.SendLogMessage(worker::LogLevel::kInfo, "Physics", "Assigning ClientData component to " + op.CallerAttributeSet[1]);
+
+        // Assign write ACL of the client data component to the worker ID received.
+        improbable::EntityAcl::Update acl_update;
+        auto current_write_acl = view.Entities[entityId].Get<improbable::EntityAcl>()->component_write_acl();
+        current_write_acl[sample::ClientData::ComponentId] = improbable::WorkerRequirementSet({{{op.CallerAttributeSet[1]}}});
+        acl_update.set_component_write_acl(current_write_acl);
+        connection.SendComponentUpdate<improbable::EntityAcl>(entityId, acl_update);
+    });
+    view.OnCommandResponse<sample::ClientData::Commands::TestCommand>([&](const worker::CommandResponseOp<sample::ClientData::Commands::TestCommand>& op) {
+        if (op.StatusCode == worker::StatusCode::kSuccess) {
+            connection.SendLogMessage(worker::LogLevel::kInfo, "Physics", "Received command response: " + std::to_string(op.Response->sum()));
+        }
+    });
+
+    int tick_count = 0;
+	float angle = 0;
+    while (is_connected) {
+        view.Process(connection.GetOpList(kGetOpListTimeoutInMilliseconds));
+
+        // Update position of entity.
+        angle += 0.5f;
+		improbable::Position::Update position_update;
```

Comment:
I can't tell if this is just the Github view, but indentation seems odd here as well.